_security/access-control/document-level-security.md

```diff
@@ -9,7 +9,10 @@ redirect_from:
 ---
 
 # Document-level security
-Document-level security lets you restrict a role to a subset of documents in an index. The easiest way to get started with document- and field-level security is to open OpenSearch Dashboards and choose **Security**. Then choose **Roles**, create a new role, and review the **Index Permissions** section, shown in the following image.
+
+Document-level security (DLS) determines the documents that a role can retrieve during read operations, such as search and get. It does not restrict write operations. If a role has permissions to index, update, or delete documents on an index, it can still modify or remove documents that are hidden by DLS. Write behavior is determined solely by index permissions and action groups.
```

Comment:
```suggestion
Document-level security (DLS) determines the documents that a role can retrieve during read operations, such as search and get. It does not restrict write operations. If a role has permissions to index, update, or delete documents in an index, it can still modify or remove documents that are hidden by DLS. Write behavior is determined solely by index permissions and action groups.
```